specification/monitor/resource-manager/microsoft.insights/2017-04-01/actionGroups_API.json

```diff
@@ -391,6 +391,20 @@
                         "$ref": "#/definitions/WebhookReceiver"
                     },
                     "description": "The list of webhook receivers that are part of this action group."
+                },
+                "ItsmReceivers": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ItsmReceiver"
+                    },
+                    "description": "The list of Itsm receivers that are part of this action group."
```

Comment:
[nit] Because "Itsm" looks so conspicuously like "Items", in the description I would either write it out or at least have it capitalized to "ITSM" to demonstrate that it is an acronym.